views/admin/template-admin-post-metabox.php

```diff
@@ -43,6 +43,12 @@
 			<?php _e( 'Hide in RSS Feed', 'whp' ); ?>
 		</label>
 	</p>
+	<p>
+		<label for='whp_hide_in_rest_api'>
+			<input type='checkbox' name="whp_hide_in_rest_api" value='1' <?php //checked( $whp_hide_in_rest_api, 1 ); ?> id='whp_hide_in_rest_api'>
```

Comment:
@conschneider You have this commented out

Reply:
Thanks for checking. 
I added a second commit. https://github.com/martinjankov/whp-hide-posts/pull/1/commits/7fb73b482e5aa393644883008224e336a16e046d that uncomments this.

Reply:
> Yea, you add change log.

✅. I will add the changelog later today.

Reply:
Changelog entry added: https://github.com/martinjankov/whp-hide-posts/pull/1/commits/766ce79e4814b93067eebec3dee608bf9d0e2ecd